### PR TITLE
feat:The source file version does not match the UI

### PR DESCRIPTION
**Pull request by kogel-net**
# Description

_Please explain the changes you've made_

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

**Comment by wzh425** [comment]
@kogel-net  Please check whether the ui interaction is consistent with figma

**Review by wzh425 — COMMENTED** [review]
@kogel-net Please check whether the ui interaction is consistent with figma

**Comment by kogel-net** [comment]
> @kogel-net Please check whether the ui interaction is consistent with figma

updated

**Comment by wzh425** [comment]
image

**Comment by wzh425** [comment]
@kogel-net  image Is it possible to align like the design

**Comment by kogel-net** [comment]
updated

**Comment by wzh425** [comment]
Please adjust your layout again

File: src/Web/Masa.Scheduler.Web.Admin/Pages/SchedulerResources/SchedulerResourceFiles.razor [hunk 1]
```diff
@@ -13,46 +13,54 @@
...
+                                                    {
+                                                        <span>@item.Version</span>
+                                                    }
+                                                </div>
+                                            </MHover>
+                                        </MCol>
+                                }
+                         }
+                        </div>
+                        <div Style="width:44px">
```
Comment (wzh425) [inline src/Web/Masa.Scheduler.Web.Admin/Pages/SchedulerResources/SchedulerResourceFiles.razor:59]: style替换为使用class ml-1

File: src/Web/Masa.Scheduler.Web.Admin/Pages/SchedulerResources/SchedulerResourceFiles.razor [hunk 2]
```diff
@@ -13,46 +13,54 @@
...
-                            </MHover>
-
-                        </MCol>
-                    }
-                </MRow>
-            </MCardText>
+                </MCardTitle>
+                <MCardText>
+                    <MRow>
+                        <div style="width:calc(100% - 44px);display: flex;flex-wrap: wrap;">
```
Comment (wzh425) [inline src/Web/Masa.Scheduler.Web.Admin/Pages/SchedulerResources/SchedulerResourceFiles.razor:32]: style display: flex;flex-wrap: wrap; 替换为class d-flex flex-wrap

File: src/Web/Masa.Scheduler.Web.Admin/Pages/SchedulerResources/SchedulerResourceFiles.razor [hunk 3]
```diff
@@ -13,46 +13,54 @@
...
+                                foreach (var item in app.Resources)
+                                {
+                                    var colIndex = resourcesIndex++;
+                                        <MCol Style="@(colIndex >= 4 && !resourceShow ? "display:none" : "")" Cols="12" Md="3">
+                                            <MHover>
+                                                <div @attributes="context.Attrs" class="@($"full-width fill-hover rounded-2 d-flex justify-center align-center regular--text h5")" style="height:119px">
+                                                    @if (context.Hover)
+                                                    {
+                                                        <MIcon Size="24" Color="error" Class="align-self-center" OnClick="()=> ShowDialog(ConfirmDialogTypes.DeleteResourceVersion, item.Id, string.Empty)">mdi-delete</MIcon>
+                                                        <span class="d-flex" style="height: 24px;padding-left:60px;padding-right:60px">
```
Comment (wzh425) [inline src/Web/Masa.Scheduler.Web.Admin/Pages/SchedulerResources/SchedulerResourceFiles.razor:44]: style padding-left:60px;padding-right:60px 替换为px-15

File: src/Web/Masa.Scheduler.Web.Admin/Pages/SchedulerResources/SchedulerResourceFiles.razor [hunk 4]
```diff
@@ -13,46 +13,54 @@
...
+                                                        <span>@item.Version</span>
+                                                    }
+                                                </div>
+                                            </MHover>
+                                        </MCol>
+                                }
+                         }
+                        </div>
+                        <div Style="width:44px">
+                            <MButton Icon OnClick="() => resourceShow = !resourceShow" Style="transform: translate(-12%, 120%);float :right;" Class="m-size--small"> <MIcon>@(resourceShow ? "mdi-chevron-down" : "mdi-chevron-up")</MIcon></MButton>
```
Comment (wzh425) [inline src/Web/Masa.Scheduler.Web.Admin/Pages/SchedulerResources/SchedulerResourceFiles.razor:60]: float :right 替换为float-right

**Review by wzh425 — COMMENTED** [review]
请调整样式用法及格式化

**Review by wzh425 — APPROVED** [review]
LGTM

**Pull request merged**